northd/northd.c

```diff
@@ -2657,6 +2660,13 @@ join_logical_ports(const struct sbrec_port_binding_table *sbrec_pb_table,
                 od->has_vtep_lports = true;
             }
 
+            if (smap_get(&nbsp->options, "enable_router_port_acl") &&
```

Comment:
Can we move this check lower, where we actually populate enable_router_port_acl?

Reply:
@dceara ok!